tests/runtests/dont-blame-interpret/PURPOSE

```diff
@@ -1,3 +1,3 @@
 PURPOSE of dont-blame-interpret
-Description: in case of crash in interpret (python, perl) abrt should blame the running script not the interpret issue#609
+Description: in case of crash in interpret (python, perl, R, tcl, php) abrt should blame the running script not the interpret issue#609
```

Comment:
I guess this incoherent mess of a sentence can also be fixed while we’re at it.

Reply:
Changed. Not sure if it's better.

Reply:
Verily better.